sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCoalesceBatches.scala

```diff
@@ -190,7 +190,7 @@ trait SplittableGoal
  */
 case object RequireSingleBatch extends CoalesceSizeGoal with RequireSingleBatchLike {
 
-  override val targetSizeBytes: Long = Long.MaxValue
+  override val targetSizeBytes: Long = 2L * 1024 * 1024 * 1024
```

Comment:
so instead of this, we'd like to change the estimation in the exec, not the targetsizeBytes here. IMHO

Reply:
Ah, yes that makes more sense. Pushed a change